src/ui/Omnibox/Omnibox.ts

```diff
@@ -55,10 +55,19 @@ export interface IOmniboxOptions extends IQueryboxOptions {
 const MINIMUM_EXECUTABLE_CONFIDENCE = 0.8;
 
 /**
- * This component is very similar to the simpler {@link Querybox} component and support all the same options/behavior except for the search-as-you-type feature.<br/>
- * In addition, it takes care of adding a type-ahead capability. The type-ahead and the suggestions it displays are customizable and extensible by any custom component.<br/>
- * The type-ahead is configurable by activating addon which are provided OOTB (facets, analytics suggestions, Reveal suggestions, and advanced Coveo syntax suggestions).<br/>
- * It is also possible for external code to provide suggestions.
+ * The Omnibox component is very similar to the simpler {@link Querybox} component. It supports all of the same options
+ * and behaviors, with the exception of the search-as-you-type feature, which it does not support.
```

Comment:
That's not true anymore (the search as you type part).

This should be removed.